docs/user/masternodes/setup-testnet.rst

```diff
@@ -1238,8 +1244,7 @@ Your masternode is now providing service on the following local ports::
   Core RPC:     19998
   Platform P2P: 26656
   Platform RPC: 26657
-  DAPI HTTP:    3000
-  DAPI gRPC:    3010
+  DAPI:    3000
```

Comment:
This is changed only on v24, while this doc representing the testnet the suppose to be running v23 network.